libs/domains/services/feature/src/lib/last-commit/last-commit.tsx

```diff
@@ -0,0 +1,87 @@
+import { type ApplicationGitRepository } from 'qovery-typescript-axios'
+import { useState } from 'react'
+import { type ServiceType } from '@qovery/domains/services/data-access'
+import { Avatar, Button, Icon, IconAwesomeEnum, Tooltip } from '@qovery/shared/ui'
+import { useCopyToClipboard } from '@qovery/shared/util-hooks'
+import { twMerge } from '@qovery/shared/util-js'
+import { useCommits } from '../hooks/use-commits/use-commits'
+
+export interface LastCommitProps {
+  className?: string
+  gitRepository: ApplicationGitRepository
+  serviceId: string
+  serviceType: Extract<ServiceType, 'APPLICATION' | 'JOB' | 'CRON_JOB' | 'LIFECYCLE_JOB'>
+}
+
+export function LastCommit({ className, gitRepository, serviceId, serviceType }: LastCommitProps) {
```

Comment:
The behavior is a little bit glitchy when you hover the component and change the icon, the width is different, should be similar to be more aesthetic. Moreover, we have a gray border on the "number of commits", it should be removed or orange

https://www.loom.com/share/156479caeef24448bf78f8bf8fea8c32

Reply:
> The behavior is a little bit glitchy when you hover the component and change the icon, the width is different, should be similar to be more aesthetic.
Fixed
https://github.com/Qovery/console/pull/865/commits/73cc81edd85353abea293a1ad00e0d4a51e34b76

> Moreover, we have a gray border on the "number of commits", it should be removed or orange
Fixed https://github.com/Qovery/console/pull/865/commits/7e1c502f0339cce7b5ba83a7fb15846ecea14449
Still ugly code unless we truly think about the inset component

Reply:
Nice thanks !

Just one thing, when we reload the page, the commit component appears after the content: https://www.loom.com/share/9a6145aabac74ffb9bd7f623b21bfdac